errors/not_found_error.py

```diff
@@ -2,4 +2,3 @@ class NotFoundError(Exception):
     def __init__(self):
         self.message = "Item not found"
         super().__init__(self.message)
-
```

Comment:
Your formatter recommends removing rows from the end of the file? Mine recommends adding it... yay formatters.

Reply:
No. Mine states a space is needed. Not sure what went down here.